packages/esm-patient-registration-app/src/config-schema.ts

```diff
@@ -306,6 +312,24 @@ export const esmPatientRegistrationSchema = {
       _default: '736e8771-e501-4615-bfa7-570c03f4bef5',
     },
   },
+  MPI: {
+    title: {
+      _type: Type.String,
+      _default: 'MPI',
+    },
+    baseAPIPath: {
+      _type: Type.String,
+      _default: '/ws/fhir2/R4/MPIPatient',
```

Comment:
This is a big red-flag for me...

1. I'm not sure we really need this as a configuration setting. This app should just work with a defined API endpoint. I don't see why an implementation would change this right now.
2. Adding a custom domain like this to the FHIR module is a non-starter. We should discuss what actual queries are necessary here and how we can properly represent those.

Reply:
This config was necessary before we agreed on a generic backend module. I'm happy to drop it. @ibacher did you have a look at: https://issues.openmrs.org/browse/HIE-1?

Reply:
@pmanko fyi. Kindly add Pawel to this thread. thanks